### ISSUE TITLE
구현 계획

### ISSUE THREAD

**Issue by emeraldgoose** [emeraldgoose/hcrot#1]
> 누구나 다 그럴싸한 계획을 갖고 있다

## 큰 범위의 구현 진행정도
- [x] MLP
- [x] #2
- [x] RNN

### MLP
- [x] Linear
- [x] CrossEntropyLoss, MSELoss
- [x] Optimizer(Gradient Descent)
### CNN
- [x] Conv2d
- [x] Flatten
- [x] ReLU
- [x] MaxPool2d
- [x] AvgPool2d
### RNN
- [x] #9
- [x] Embedding
- [x] LSTM

**Comment by emeraldgoose** [comment]
Numpy 없이 구현하기 위해 Numpy 함수들까지 같이 구현했지만 코드 가독성이 너무 안좋아지므로 Numpy를 사용한 코드로 리팩토링 진행중
